authoring-new-actions.md

```diff
@@ -0,0 +1,130 @@
+# Process to publish GitHub Actions for Azure
+
+# Introduction
+
+[GitHub Actions](https://help.github.com/en/articles/about-github-actions) gives developers the flexibility to build an automated software development lifecycle workflow directly from their repositories. Since releasing Actions in November 2017, GitHub partners have contributed enormously to Marketplace, so teams can extend and automate workflows with their existing tools.
+
+Microsoft [announced](https://azure.microsoft.com/en-us/blog/github-actions-for-azure-is-now-generally-available/) the general availability of [GitHub Actions for Azure](https://github.com/Azure/actions) for creating workflows to package, release, and deploy apps to the cloud from GitHub. [GHActions\_vTeam](mailto:GHActions_vTeam@microsoft.com) has been formed to spearhead this effort of collaborating with various product teams at Microsoft to release these actions.
+
+Also, at [Microsoft Build 2020](https://aka.ms/MS-build-session-Continuous-Delivery) we announced that GitHub Actions for Azure are now integrated into Visual Studio Code, Azure CLI, and the Azure Portal simplifying the experience of [deploying to Azure from your favorite tools.](https://azure.microsoft.com/en-us/blog/deploy-to-azure-using-github-actions-from-your-favorite-tools/)
+
+This document is intended to help Microsoft teams and developers with the recommended process to create and publish more actions to GH marketplace. Please reach out to the [vTeam](mailto:GHActions_vTeam@microsoft.com) for any queries.
+
+# Actions Plan
+
+[https://github.com/Azure/actions](https://github.com/Azure/actions) has the complete list of all the actions that we have released to Marketplace. To help GH developers easily create GitHub CI/CD workflows targeting Azure, we also published [Azure starter templates](https://github.com/Azure/actions-workflow-samples) to deploy apps created with popular languages and frameworks such as .NET, Node.js, Java, PHP, Ruby or Python, in containers or running on any operating system.
+
+We released Actions for working with a variety of Azure services, from Web Apps to serverless Functions, as well as Azure SQL and MySQL databases. We also released Actions for building and deploying container-based applications that work with Docker and Kubernetes on any environment and cloud, in addition to their managed Azure Kubernetes Service.
+
+Here is the list of partner teams within Microsoft that we are collaborating:
+
+| **Azure Service** | **Actions** | **Point of contact from Product team** |
```

Comment:
Is this list expected to be public? 

Also, do the PoCs currently own any actions from our azure/actions roster?

Reply:
I removed this section. We can discuss offline if we need to publish such a roster in public. Even if we do, we could omit emails and just include team-handles of individual Azure teams.